credoai/modules/dataset_modules/dataset_base.py

```diff
@@ -68,91 +95,135 @@ def _group_differences(self):
         return diffs
     
     def _run_cv(self, pipe):
+        """Determines cross-validated ROC-AUC score
```

Comment:
This docstring should mention "redundant encoding". That's really what this is testing for

https://towardsdatascience.com/programming-fairness-in-algorithms-4943a13dd9f8

Reply:
docstring updated